src/Sylius/Bundle/WebBundle/Resources/views/Backend/Macros/misc.html.twig

```diff
@@ -41,6 +41,15 @@
     {% else %}
         {% set class = 'label-danger' %}
     {% endif %}
+{% elseif type == 'shipment' %}
+    {% set label = 'sylius.shipment.state.'~state %}
+    {% if state is same as('shipped') %}
```

Comment:
I got the idea of using `Strict Equality` by looking at the code above this part which was checking for a particular payment state. Do you think we should use `Strict Equality` here?